server/src/main/java/com/cloud/api/query/dao/StoragePoolJoinDaoImpl.java

```diff
@@ -229,6 +231,17 @@ public StoragePoolResponse newStoragePoolForMigrationResponse(StoragePoolJoinVO
         poolResponse.setDiskSizeTotal(pool.getCapacityBytes());
         poolResponse.setDiskSizeAllocated(allocatedSize);
         poolResponse.setCapacityIops(pool.getCapacityIops());
+
+        if (storagePool != null) {
+            poolResponse.setManaged(storagePool.isManaged());
+            if (storagePool.isManaged()) {
+                DataStore store = dataStoreMgr.getDataStore(pool.getId(), DataStoreRole.Primary);
+                PrimaryDataStoreDriver driver = (PrimaryDataStoreDriver) store.getDriver();
+                long usedIops = driver.getUsedIops(storagePool);
+                poolResponse.setAllocatedIops(usedIops);
```

Comment:
this code is sync with the response parameters set for managed pools here:

https://github.com/apache/cloudstack/blob/674129cd588e9be67923fc06d652170f405f2d07/server/src/main/java/com/cloud/api/query/dao/StoragePoolJoinDaoImpl.java#L147